lib/runtime-c-api/src/import/wasi.rs

```diff
@@ -44,21 +56,74 @@ pub unsafe extern "C" fn wasmer_wasi_generate_import_object(
     let mapped_dir_list = get_slice_checked(mapped_dirs, mapped_dirs_len as usize);
 
     wasmer_wasi_generate_import_object_inner(
+        Version::Snapshot1,
```

Comment:
In order to avoid potential regressions in a close future (if a new WASI version is published and we forgot to update the C API), I've created https://github.com/wasmerio/wasmer/pull/1029 that introduces the `Latest` version. This is a nice usecase for this new “floating” version.

Reply:
Applied.